src/sign-in-xplat/MainPage.xaml.cs

```diff
@@ -5,6 +5,8 @@
 using System.Net.Http;
 using System.Net.Http.Headers;
 using System.Net.Http.Json;
+using System.Text.Encodings.Web;
+using System.Text.Json;
 using System.Threading.Tasks;
 using Microsoft.Graph;

```

Comment:
Since you are no longer deserializing to the MSAL User object, can we remove the direct reference?  Like you did in the other example recently?  Or is this still needed?

remove -->	  `<PackageReference Include="Microsoft.Graph" Version="4.11.0" />`
leave -->	  `<PackageReference Include="Microsoft.Identity.Client" Version="4.38.0" />`

Reply:
good catch @ckittel 

done |  addressed from de2a9951d7e42d46a3b5d318d40683489e7c4de2